tests/jest/ext.wikibase.facetedsearch.test.js

```diff
@@ -1,19 +1,62 @@
+const actual = require( '../../resources/ext.wikibase.facetedsearch.js' );
+
 describe( 'ext.wikibase.facetedsearch.js', () => {
```

Comment:
Does this describe block make sense? Isn't there already grouping based on in which file the tests reside?

It is often nice to group per function being tested. So that would be an additional group for say `getListFacetQuerySegments`

Reply:
Followed up in #141